src/graph/entities/edge.c

```diff
@@ -73,3 +73,10 @@ void Edge_Free(Edge* edge) {
 	if(edge->relationship != NULL) free(edge->relationship);
 	free(edge);
 }
+
+void Edge_Print(const Edge* edge, FILE *out) {
+    if(!out) out = stdout;
+    GraphEntity_Print((GraphEntity*) edge, GETYPE_EDGE, 0);
+    if (edge->relationship || edge->alias)
```

Comment:
The intention here was to print if either values is available. A null value is printed as "(null)". Although this is diagnostic code, we can add code to print empty strings instead of nulls.